pkg/disk/disk.go

```diff
@@ -67,14 +67,23 @@ const (
 	// Partition type ID for any native Linux filesystem on dos
 	DosLinuxTypeID = "83"
 
-	// Partition type ID for BIOS boot partition on dos
-	DosBIOSBootID = "ef02"
+	// Partition type ID for LVM on dos
+	DosLVMTypeID = "8e"
```

Comment:
Maybe for a followup, let's use this new constant everywhere, right now I still see:
```console
$ git grep '"8e"'
cmd/otk/osbuild-gen-partition-table/main_test.go:                                                       Type:  "8e",
pkg/disk/partition_table.go:                    part.Type = "8e"
```